spanner/google/cloud/spanner_v1/client.py

```diff
@@ -95,9 +95,17 @@ class Client(ClientWithProject):
                         client. If not provided, defaults to the Google
                         Application Default Credentials.
 
+    :type client_info: :class:`google.api_core.client_info.ClientInfo`
```

Comment:
Do we need the gapic version?

Same comment as https://github.com/googleapis/google-cloud-python/pull/7876#discussion_r281872712

Reply:
26c5dec